Khan/khan.github.io
# Add aphrodite

Summary: Adds aphrodite (and it's snazzy logo from Elizabeth Lin!) to
the KA open source page.

Feedback wanted on the description. Suggestions: "CSS for Robots"

Test Plan:
 - Run a web server in this directory
 - Visit the page, see that Aphrodite shows up
 - Scroll to the bottom, see that all of the rows look fine.

![image](https://cloud.githubusercontent.com/assets/217900/20329412/d551de42-ab4c-11e6-9111-c937842fc9b7.png)

@jdan
